src/components/UsersDisplay/UsersDisplay.js

```diff
@@ -0,0 +1,97 @@
+import { Grid, Paper, Typography, Button } from "@mui/material";
+import axios from "axios";
+import { useEffect, useState } from "react";
+import { isEmpty } from "lodash";
+import { useContext } from "react";
+import AppContext from "../../context/AppContext";
+
+const UsersDisplay = ({ kitchenId }) => {
+  const [users, setUsers] = useState([]);
+  const { loggedInUser } = useContext(AppContext);
+  const fetchUserList = async () => {
+    const usersList = await axios.get(`/v1/kitchen/${kitchenId}/users`);
+    setUsers(usersList.data);
+  };
+
+  const matchingUser = users.find((user) => user.id === loggedInUser.id);
+
+  // Fetches users on intial startup.
+  useEffect(() => {
+    fetchUserList();
+  }, []);
+  return (
+    // One Parent Element
+    <Grid width="100%">
```

Comment:
If this Grid is the parent, it should be a "Grid container". 

```suggestion
    <Grid container>
```